pkg/retry/retry.go

```diff
@@ -38,6 +39,7 @@ func Run(initialInterval time.Duration, maxRetries uint64, f func() error) error
 	}, retryCfg)
 }
 
+// RunWithInfiniteRetry is Deprecated, try to use instead Do WithInfiniteTries option instead.
 // RunWithInfiniteRetry retries the specified function indefinitely, until a backoff.PermanentError is encountered.
 // notifyFunc will be called each time before sleeping with the total elapsed time.
 func RunWithInfiniteRetry(initialInterval time.Duration, f func() error, notifyFunc func(elapsed time.Duration)) error {
```

Comment:
Can remove this function directly?

Reply:
to be safe, will deleted after this pr merged to master, just in case have new code added with this func during the review process, like in https://github.com/pingcap/ticdc/pull/1928/commits/b070ea28f0f439f56704559bf3090e6767f9ef58#diff-678290e01edfa8888e61fc247ea83b5a574c52f5c9e4516769c2b26ea63198c1L328